  8.3% (1/12) 2016_12_02_0101_s03_01_g8b.png reg3/2016_12_02_0101_s03_01_g8b.tif plain copy
 16.7% (2/12) reg3/2016_12_02_0101_s03_01_g8b.tif 2016_12_02_0101_s03_02_g8b.png 199 83.540355 ['83.540355'] 83.540355
 25.0% (3/12) reg3/2016_12_02_0101_s03_02_g8b.tif 2016_12_02_0101_s03_03_g8b.png 199 22.463305 ['22.463305'] reg3/2016_12_02_0101_s03_02_g8b.log//TransformParameters.0.txt 199 22.463305 ['22.463305'] 22.463305
 33.3% (4/12) reg3/2016_12_02_0101_s03_03_g8b.tif 2016_12_02_0101_s03_04_g8b.png 199 25.63025 ['25.630250'] reg3/2016_12_02_0101_s03_03_g8b.log//TransformParameters.0.txt
Traceback (most recent call last):
  File "/elastix_scripts/recRegStack.py", line 309, in <module>
    main()
  File "/elastix_scripts/recRegStack.py", line 101, in main
    register(FNs[start:], FNo, args, FNp) # skip upto start
  File "/elastix_scripts/recRegStack.py", line 258, in register
    selx[i].Execute()
  File "/opt/SimpleElastix/lib/python/SimpleITK/SimpleITK.py", line 10996, in Execute
    return _SimpleITK.ElastixImageFilter_Execute(self)
RuntimeError: Exception thrown in SimpleITK ElastixImageFilter_Execute: /SimpleElastix/Code/Elastix/src/sitkElastixImageFilterImpl.cxx:259:
sitk::ERROR: 
itk::ExceptionObject (0x562f57d56c00)
Location: "unknown" 
File: /selx_build/Elastix/Core/Main/elxElastixFilter.hxx
Line: 253
Description: itk::ERROR: Self(0x562f583c3870): Internal elastix error: See elastix log (use LogToConsoleOn() or LogToFileOn()).


